src/mrack/providers/beaker.py

```diff
@@ -63,13 +62,15 @@ def __init__(self):
         self._name = PROVISIONER_KEY
         self.conf = PyConfigParser()
         self.poll_sleep = 30  # seconds
+        self.keypair = None
 
-    async def init(self, prov_config):
+    async def init(self, max_attempts, reserve_duration, keypair):
```

Comment:
:+1:

Reply:
I am adding another status because queued seems to be missing and causes failure when beaker is busy.